Models/PhotosynthesisModel.cs

```diff
@@ -0,0 +1,229 @@
+﻿using System;
+using System.Linq;
+
+using DCAPST.Canopy;
+using DCAPST.Interfaces;
+
+namespace DCAPST
+{
+    public class PhotosynthesisModel
+    {
+        public ISolarGeometry Solar { get; set; }
+        public ISolarRadiation Radiation { get; set; }
+        public ITemperature Temperature { get; set; }
+        public ITotalCanopy Canopy { get; set; }
+
+        /// <summary>
+        /// Biochemical Conversion & Maintenance Respiration
+        /// </summary>
+        public double B { get; set; } = 0.409;
```

Comment:
It's called B because that's what it is in the literature, as per the [appendix](https://www.publish.csiro.au/FP/acc/FP17225/FP17225_AC.pdf). Wasn't sure which way to go with it.

Units are also listed there, and will be added.

Reply:
Naming things is definitely hard. 
Bcmr should work ok.
Definitely add the units where they are known.